### PR TITLE
fix (webapi): loading of ontologies on startup

### LINKED ISSUE #1154: OntologyCache empty after startup
When I start webapi and access the route `/v2/ontologies/metadata` I get:

> {
"error": "org.knora.webapi.ApplicationCacheException: The Knora API server has not loaded any ontologies, perhaps because of an invalid ontology"
}

When I start webapi, it is up immediately. Before it took a couple of seconds.

> webapi [2019-01-10 16:06:31,539] INFO  - ApplicationStateActor - appStateChanged - to state: StartingUp
webapi [2019-01-10 16:06:31,539] INFO  - ApplicationStateActor - appStateChanged - to state: WaitingForRepository
webapi [2019-01-10 16:06:31,595] INFO  - ApplicationStateActor - appStateChanged - to state: RepositoryReady
webapi [2019-01-10 16:06:31,595] INFO  - ApplicationStateActor - appStateChanged - to state: CreatingCaches
webapi [2019-01-10 16:06:31,738] INFO  - ApplicationStateActor - appStateChanged - to state: CachesReady
webapi [2019-01-10 16:06:31,738] INFO  - ApplicationStateActor - appStateChanged - to state: LoadingOntologies
webapi [2019-01-10 16:06:31,739] INFO  - ApplicationStateActor - appStateChanged - to state: OntologiesReady
webapi [2019-01-10 16:06:31,739] INFO  - ApplicationStateActor - appStateChanged - to state: Running
webapi [2019-01-10 16:06:31,739] INFO  - ApplicationStateActor - 
webapi ================================================================
webapi Knora API Server started at http://0.0.0.0:3333
webapi ----------------------------------------------------------------
webapi DB-Name: knora-test
webapi DB-Type: graphdb-se
webapi DB Server: localhost, DB Port: 7200
webapi ================================================================
webapi

### PR DESCRIPTION
**Pull request by subotic**
### Issues

- resolves #1154

### CODE REVIEW

**Comment by tobiasschweizer** [comment]
ok, trying it now ...

File: webapi/src/main/scala/org/knora/webapi/Main.scala [hunk 1]
```diff
@@ -76,7 +76,7 @@ object Main extends App with LiveCore with KnoraService {
             """.stripMargin)
     } else {
         /* Start the HTTP layer, allowing access */
-        startService(true)
+        startService(false)
```
Comment (tobiasschweizer) [inline webapi/src/main/scala/org/knora/webapi/Main.scala]: Could you add a comment why this needs to be false?

**Comment by subotic** [comment]
thanks for the review and sorry for breaking it in the first place.

File: webapi/src/main/scala/org/knora/webapi/Main.scala (hunk 1 above)
Comment (subotic) [reply webapi/src/main/scala/org/knora/webapi/Main.scala]: done

**Comment by tobiasschweizer** [comment]
> thanks for the review and sorry for breaking it in the first place.

you are welcome. And don't worry: shit happens :-)

**Pull request merged**